### PR TITLE
Set Clang format ColumnLimit to 120

### PR DESCRIPTION
**Pull request by vmoroz**
Last few PRs have a lot of code changes mostly because we had to auto-format the code.
In this PR we format existing code files and set the Clang format ColumnLimit to 120 as in the react-native-windows projects.
It would be ideal if this PR is merged before the other two outstanding PRs to reduce number of changed lines in these PRs.
There are no code changes in PR besides the auto-formatting and fixing some spelling issues in a few comments.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/v8-jsi/pull/52)

### CODE REVIEW

File: src/testmain.cpp [hunk 1]
```diff
@@ -16,7 +16,8 @@ std::vector<facebook::jsi::RuntimeFactory> runtimeGenerators() {
   }};
 }
 
-}}
```
Comment (NikoAri) [inline src/testmain.cpp:19]: (nit) could we use modern style namespace::syntax?

File: src/V8Platform.h [hunk 2]
```diff
@@ -59,16 +59,11 @@ class ETWTracingController : public v8::TracingController {
       unsigned int flags,
       int64_t timestamp) override;
 
-  void UpdateTraceEventDuration(
-      const uint8_t *category_enabled_flag,
```
Comment (NikoAri) [inline src/V8Platform.h:63]: > co [](start = 5, length = 3)

(nit) usually, having separate line per argument for long arguments is more readable

**Review by NikoAri — APPROVED** [review]
:shipit:

File: src/V8Platform.h (hunk 2 above)
Comment (vmoroz) [reply src/V8Platform.h:63]: This is an automatic formatting. We better keep it "as-is".

**Pull request merged**